packages/node-fhir-server-core/src/server/server.js

```diff
@@ -200,7 +200,8 @@ class Server {
 
   configurePassport() {
     if (this.config.auth && this.config.auth.strategy) {
-      let { strategy } = require(path.resolve(this.config.auth.strategy.service));
+      const service = this.config.auth.strategy.service;
```

Comment:
makes more sense to make it into a separate function

const Strategy = require('passport-http-bearer').Strategy;
  configurePassport(strategy) {
    if(strategy instanceof passport.Strategy){
      passport.use(strategy);
      return true;
    }
    return false;
  }

Reply:
Thanks for checking, I will modify it, but I am not sure I understand entirely. Do you mean something like this? What to do with the return value?

```javascript
configurePassport() {
  if (this.config.auth && this.config.auth.strategy) {
    const service = this.config.auth.strategy.service;
    let { strategy } = typeof service === 'string' ? require(path.resolve(service)) : service;
    this.configurePassportWithStrategy(strategy);
  }

  // return self for chaining
  return this;
}

configurePassportWithStrategy(strategy) {
  if (strategy instanceof passport.Strategy) {
    passport.use(strategy);
    return true;
  }
  return false;
}
```

Reply:
don't update configurePassport().  Just add the new configurePassportWithStrategy() and use that instead of configurePassport().

Reply:
ok, but I think that will not be very useful - those who may need it will not be able to use the default `initialize`, but instead have to do custom initialization. In such case the users can just skip `configurePassport()` and call `passport.use()` directly.

I copied the code from `verifyAndLoadProfiles` which already supports both strings and objects, so this change doesn't break existing behavior, just adds more possibility and unification.